recipes/asyncio-helpers/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "asyncio-helpers" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sci-bots/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: ee1c3b674fb58b3362c886290601007f08ef11db4119688d80771ec0cc0ec696
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: true  # [py>36]
+
+requirements:
+  host:
+    - python
```

Required for `noarch`:
```suggestion
    - python >=2.7,<=3.6
```